changelog/18.0/18.0.0/summary.md

```diff
@@ -5,6 +5,8 @@
 - **[Major Changes](#major-changes)**
   - **[Breaking Changes](#breaking-changes)**
     - [Local examples now use etcd v3 storage and API](#local-examples-etcd-v3)
+  - **[Deprecations and Removals](#deprecations-and-removals)**
+    - [Legacy Clients](#legacy-clients)
```

Comment:
We already have a `Deprecations and Deletions` section, I think let's move `Legacy Clients` to that already existing section.

Reply:
Good catch! Somehow I missed the existing section.